examples/inverse/plot_lcmv_beamformer.py

```diff
@@ -69,8 +69,10 @@
 colors = ['b', 'k', 'r']
 
 for pick_ori, name, desc, color in zip(pick_oris, names, descriptions, colors):
+    # compute unit-noise-gain beamformer with whitening of the leadfield and
+    # data (enabled by passing a noise covariance matrix)
     stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.05,
-               pick_ori=pick_ori)
+               pick_ori=pick_ori, weight_norm='unit-noise-gain')
```

Comment:
If I use the channel selection in both, the output is still identical - we get arbitrary activity in areas which are not covered by channels in both, here a volume plot (threshold was not changed and shows only auditory activity with full sensor coverage):

![image](https://user-images.githubusercontent.com/20129119/28116650-3cb697c0-670a-11e7-94ac-8eb9487371bd.png)